specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/disasterRecoveryConfigs.json

```diff
@@ -425,13 +425,13 @@
               "type": "string",
               "description": "ARM Id of the Primary/Secondary eventhub namespace name, which is part of GEO DR pairing"
             },
-            "type": {
+            "replicationType": {
```

Comment:
Why are you changing type to replicationType ?

Reply:
There was Some conflict error with type name
When I Further investigate It is saying to rename that parameter.

Reply:
The rename will cause inconsistency between ARM template and swagger. It is not a reasonable change.